3_javascript/javascript2/week1/script.js

```diff
@@ -0,0 +1,65 @@
+// JavaScript
+document.getElementById('generateButton').addEventListener('click', function () {
+    const name = document.getElementById('nameInput').value;
+    // Call a function to generate and display the spirit animal name
+    generateSpiritAnimalName(name);
+});
+
+const spiritAnimals = [
+    "The Wise Owl",
+    "The Brave Lion",
+    "The Curious Fox",
+    "The Playful Dolphin",
+    "The Strong Bear",
+    "The Graceful Gazelle",
+    "The Gentle Panda",
+    "The Free Spirit Horse",
+    "The Mystical Wolf",
+    "The Colorful Peacock"
+];
+
+function generateSpiritAnimalName(name) {
+    const randomSpiritAnimal = spiritAnimals[Math.floor(Math.random() * spiritAnimals.length)];
+    const spiritAnimalName = name + " - " + randomSpiritAnimal;
+    document.getElementById('spiritAnimalName').textContent = "Spirit Animal Name: " + spiritAnimalName;
+}
+
+
+document.getElementById('newAnimalButton').addEventListener('click', function () {
```

Comment:
This formulation is not wrong, but it is not the most readable. 
Would be perhaps nice to declare the elements as variables at the top of the documents first and then add the event listeners as the next step 🙏🏽